### PR TITLE
quincy: doc/cephfs: add "metrics" to left pane

### PR DESCRIPTION
**Pull request by zdover23**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/dev/perf_counters.rst [hunk 1]
```diff
@@ -200,3 +202,50 @@ The actual dump is similar to the schema, except that average values are grouped
    }
  }
 
+.. _Labeled Perf Counters:
+
+Labeled Perf Counters
```
Comment (idryomov) [inline doc/dev/perf_counters.rst:207]: This should be dropped -- labeled perf counters appeared in reef.

**Review by idryomov — COMMENTED** [review]
I'll let @ceph/cephfs handle this, but I suspect this entire PR doesn't apply to quincy.

File: doc/dev/perf_counters.rst (hunk 1 above)
Comment (zdover23) [reply doc/dev/perf_counters.rst:207]: Roger that, @idryomov. I'll drop it now.

**Review by joscollin — CHANGES_REQUESTED** [review]
You need to cherry-pick the changes from https://github.com/ceph/ceph/pull/57628 instead.

File: doc/dev/perf_counters.rst (hunk 1 above)
Comment (zdover23) [reply doc/dev/perf_counters.rst]: ... and it's dropped. "Labeled Perf Counters" has been removed from the Quincy docs.

Note to near-future Zac: Add a note to the Reef and Squid docs informing the reader that "Labeled Perf Counters" are new in Reef.

**Comment by joscollin** [comment]
@zdover23 The backport of https://tracker.ceph.com/issues/66173 is not for `quincy` because the feature https://tracker.ceph.com/issues/64387 is only on reef and squid. So you could close this PR.

**Pull request closed without merging**